opentok/opentok.py

```diff
@@ -522,6 +509,31 @@ def signal(self, session_id, payload, connection_id=None):
         else:
             raise RequestError("An unexpected error occurred", response.status_code)
 
+    def get_stream(self, session_id, stream_id):
+        """
+        Returns an Stream object that contains information of an OpenTok stream:
+        
+        -id: The stream ID
+        -videoType: "camera" or "screen"
+        -name: The stream name (if one was set when the client published the stream)
+        -layoutClassList: It's an array of the layout classes for the stream
+        """
+        endpoint = self.endpoint.get_stream_url(session_id, stream_id)
+        response = requests.get(
+            endpoint, headers=self.json_headers(), proxies=self.proxies, timeout=self.timeout
+        )
+
+        if response.status_code == 200:
+            return Stream(response.json())
+        elif response.status_code == 400:
+            raise GetStreamError('Invalid request.')
```

Comment:
I think we should add more details to this error: 

```
 Invalid request. This response may indicate that data in your request data is invalid JSON. Or it may indicate that you do not pass in a session ID or you passed in an invalid stream ID.
```